skmatter/sample_selection/_base.py

```diff
@@ -399,6 +399,15 @@ def __init__(
         )
 
 
+def directional_distance(equations, points):
```

Comment:
Would be nice if we add here explanation for math. I did not find this trivial at all